content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -25,7 +25,13 @@ it succeeds, making sure those probes don't interfere with the application start
 This can be used to adopt liveness checks on slow starting containers, avoiding them
 getting killed by the kubelet before they are up and running.
 
+{{< caution >}}
+Liveness probes can be a powerful way to recover from application failures, but they should be used with caution. Liveness probes must be configured carefully to ensure that they truly indicate unrecoverable application failure, for example a deadlock. A common pattern for liveness probes is to use the same low-cost HTTP endpoint as for readiness probes, but with a higher failureThreshold. This ensures that the pod is observed as not-ready for some period of time before it is hard killed.
+{{< /caution >}}
 
+{{< note >}}
+Incorrect configuration of liveness probes can lead to cascading failures. For example, restarting the container when it has high load, as opposed to being crashed, can lead to failed client requests or traffic being shifted onto other pods in the same deployment or service, thereby overloading them.
```

Comment:
The syntax around "as opposed to being crashed" needs a fix.

This, again, is misleading users that liveness probes are used for determining readiness.

Reply:
May you please suggest me on what to write instead of "as opposed to being crashed"?

Reply:
I cannot make a suggestion because I don't understand what you are trying to say.
We can start by writing shorter sentences and then optimize.

Reply:
Sorry @tengqm for not communicating my problem in more understandable way.
I've edited down some the lines:

"Incorrect configuration of liveness probes can lead to cascading failures. For example, killing the container when it has high load causes excessive downtime and lead to failed client request."

May you please review this?

Reply:
Taking into account the remarks, here are some ideas :

- An incorrect implementation of liveness probes can result in cascading failures, including the container being restarted when it is under high load. 

- This can make your application less scalable, resulting in failures in client requests. 

- Moreover, if some pods fail due to these issues, the remaining pods may experience an increased workload as they handle the redirected traffic.

Reply:
@binboum & @tengqm may you both please review this:

**_Incorrect implementation of liveness probes can lead to cascading failures. This results in restarting of container under high load; failed client requests as your application became less scalable; and increased workload on remaining pods due to some failed pods._**

Reply:
This revision is a lot clearer to me.